Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -84,6 +84,73 @@ scoped[Simplicial] notation3 "Δ[" n "]" => SSet.standardSimplex.obj (SimplexCat
 instance : Inhabited SSet :=
   ⟨Δ[0]⟩
 
+section Product
+
+/-- `Prod X Y` is the explicit binary product of simplicial sets `X` and `Y`. -/
+def Prod (X Y : SSet) : SSet where
```

Comment:
```suggestion
def prod (X Y : SSet) : SSet where
```
As the product is a term of type `SSet` rather than a type itself (do not worry too much if what I am saying does not immediately make sense), the naming convention is that the name should start with a small letter.
The API around `Prod.` should be renamed accordingly.

Reply:
That makes sense, I wasn't aware of that convention though. Thanks!